product_docs/docs/biganimal/release/overview/03_security/compliance_and_certifications.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Security compliance and certifications
+navTitle: Compliance and certifications
+---
+
+BigAnimal adheres to the following security standards, certifications, and capabilities: 
+
+## SOC 2 Type I
```

Comment:
I would combine 'SOC 2 Type I' and 'SOC 2 Type II' under 'SOC 2' as a single category and then explain each of them. Here is a good reference point. https://www.mongodb.com/cloud/trust/compliance/soc

Reply:
Should the current content be removed and rewritten, or can some of it just be reorganized?